docs/pages/guides/upgrading-to-v3.mdx

```diff
@@ -79,7 +79,7 @@ That's why we recreated it from scratch.
 Now `onChange` prop for `<Keyboard(Date|Time)Picker />` has different signature from the pure pickers:
 
 ```ts
-type KeybordOnChange = (date: MaterialUiPickersDate | null, value: string | undefined) => void;
+type KeybordOnChange = (date: DateType | null, value: string | undefined) => void;
```

Comment:
I'm not sure we should change the type in this pull request. I know @dmtrKovalenko Is working on the problem. What about we focus on the test for now? :)

```suggestion
type KeybordOnChange = (date: MaterialUiPickersDate | null, value: string | undefined) => void;
```

Reply:
Sounds good!